ui/components/multichain/account-list-item/account-list-item.js

```diff
@@ -444,19 +447,28 @@ const AccountListItem = ({
             </Box>
           )}
         </Box>
-        {accountLabel ? (
-          <Tag
-            label={accountLabel}
-            labelProps={{
-              variant: TextVariant.bodyXs,
-              color: Color.textAlternative,
-            }}
-            startIconName={
-              account.metadata.keyring.type === KeyringType.snap
-                ? IconName.Snaps
-                : null
-            }
-          />
+        {accountLabels.length > 0 ? (
+          <Box flexDirection={FlexDirection.Row}>
+            {accountLabels.map((label) => {
+              return (
+                <Tag
+                  data-testid={`account-list-item-tag-${account.id}-${label}`}
+                  key={label}
+                  label={label}
+                  labelProps={{
+                    variant: TextVariant.bodyXs,
+                    color: Color.textAlternative,
+                  }}
+                  startIconName={
+                    account.metadata.keyring.type === KeyringType.snap &&
+                    !isSnapPreinstalled(account.metadata?.snap?.id)
+                      ? IconName.Snaps
+                      : null
+                  }
```

Comment:
IDK if that's intended, but I think we want to add the `IconName.Snaps` icon only for the "Snap name" pill... But here, is we have a non-preinstalled Snap that uses a different entropy source, we would show 2 pills:
1. A pill with the Snap name
2. A pill with `SRP #N` with a Snap icon

Reply:
We will add only 1, notice the break on the switch, it' always one